### PR TITLE
chore(scheduling): Added revert chaos 👾👾

### PR DESCRIPTION
**Pull request by S-ayanide**
Signed-off-by: Sayan Mondal <sayan@chaosnative.com>

## Proposed changes

* Added Revert Chaos

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx [hunk 1]
```diff
@@ -84,13 +89,52 @@ const WorkflowTable = forwardRef(({ isCustom }: WorkflowTableProps, ref) => {
...
+    parsed(manifest).forEach((engine) => {
+      deleteEngines = `${deleteEngines + engine} `;
+    });
+
+    deleteEngines += '-n {{workflow.parameters.adminModeNamespace}}';
+
+    parsedYAML.spec.templates[parsedYAML.spec.templates.length] = {
+      name: 'revert-chaos',
+      container: {
+        image: 'litmuschaos/k8s:latest',
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx:113]: this is a constant i guess which may be declared in some file or at the variable declaration part above. @S-ayanide

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx [hunk 2]
```diff
@@ -111,54 +156,104 @@ const WorkflowTable = forwardRef(({ isCustom }: WorkflowTableProps, ref) => {
...
+          <TableContainer className={classes.table} component={Paper}>
+            <Table aria-label="simple table">
+              <TableHead>
+                <TableRow>
+                  <TableCell>Sequence</TableCell>
+                  <TableCell align="left">Name</TableCell>
+                  <TableCell align="left">Namespace</TableCell>
+                  <TableCell align="left">Application</TableCell>
+                  <TableCell align="left">Probes</TableCell>
+                </TableRow>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx:169]: add translations

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx [hunk 3]
```diff
@@ -111,54 +156,104 @@ const WorkflowTable = forwardRef(({ isCustom }: WorkflowTableProps, ref) => {
...
+                        {experiment.Application}
+                      </TableCell>
+                      <TableCell align="left">{experiment.Probes}</TableCell>
+                    </TableRow>
+                  ))
+                ) : (
+                  <TableRow>
+                    <TableCell colSpan={5}>
+                      <Typography align="center">
+                        Please add experiments to see the data
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx:202]: add translation

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx [hunk 4]
```diff
@@ -111,54 +156,104 @@ const WorkflowTable = forwardRef(({ isCustom }: WorkflowTableProps, ref) => {
...
+                  </TableRow>
+                )}
+              </TableBody>
+            </Table>
+          </TableContainer>
+          {isCustom && (
+            <TableContainer className={classes.revertChaos} component={Paper}>
+              <Row className={classes.wrapper}>
+                <div className={classes.key}>
+                  <Typography>Revert Schedule</Typography>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx:214]: add translation

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx [hunk 5]
```diff
@@ -111,54 +156,104 @@ const WorkflowTable = forwardRef(({ isCustom }: WorkflowTableProps, ref) => {
...
-                    <Typography align="center">
-                      Please add experiments to see the data
-                    </Typography>
-                  </TableCell>
-                </TableRow>
-              )}
-            </TableBody>
-          </Table>
-        </TableContainer>
+                      True
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx:235]: translation

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx [hunk 6]
```diff
@@ -111,54 +156,104 @@ const WorkflowTable = forwardRef(({ isCustom }: WorkflowTableProps, ref) => {
...
+                        backgroundColor: !revertChaos
+                          ? theme.palette.error.main
+                          : theme.palette.disabledBackground,
+                        color: !revertChaos
+                          ? theme.palette.common.white
+                          : theme.palette.text.disabled,
+                      }}
+                      aria-label="centered"
+                    >
+                      False
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx:249]: translation

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
some translations and one constant declaration required.

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx (hunk 1 above)
Comment (S-ayanide) [reply litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx:115]: This is a part of the YAML, thus better not to have it in constants.

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Review by amityt — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**